tracee-ebpf/tracee/write_capture.go

```diff
@@ -178,7 +178,7 @@ func (t *Tracee) processFileWrites() {
 			// Rename the file to add hash when last chunk was received
 			if meta.BinType == sendKernelModule {
 				if uint64(meta.Size)+meta.Off == kernelModuleMeta.Size {
-					fileHash := getFileHash(fullname)
+					fileHash, _ := computeFileHash(fullname)
```

Comment:
What about errors? In other places we call `t.handleError(err)`

Reply:
Previous to this PR change if the value returned by `fileHash` was `""` then under the hood we had an error. The case in which `getFileHash` returns an empty string is not handled so far. I'm open to suggestions here.

Reply:
Looking at the next line it seems that a valid hash is expected to append it to the fullname:

```go
fileHash, _ := computeFileHash(fullname)
os.Rename(fullname, fullname+"."+fileHash)
```

Reply:
right, but if we have an error we would append an empty string, not sure this is something intended though.